101-hdinsight-linux-with-edge-node/azuredeploy.json

```diff
@@ -88,7 +87,7 @@
               "name": "[replace(replace(reference(resourceId('Microsoft.Storage/storageAccounts', variables('clusterStorageAccountName')), '2016-01-01').primaryEndpoints.blob,'https://',''),'/','')]",
               "isDefault": true,
               "container": "[parameters('clusterName')]",
-              "key": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('clusterStorageAccountName')), '2016-01-01').key1]"
+              "key": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('clusterStorageAccountName')), '2015-05-01-preview').key1]"
```

Comment:
I think you want to use the same apiVersion as the one you're provisioning... there might be a breaking change between those versions and listKeys so something like:

listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('clusterStorageAccountName')), '2016-01-01').keys[0].value)